deploy/rbac-permissions-operator/00-dedicated-admin.Namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dedicated-admin
```

Comment:
I still didn't get why we're deploying this `dedicated-admin` here. Since all the objects in that PR are under `openshift-rbac-permissions`